src/main/java/org/mifos/connector/mockpaymentschema/service/AirtelMockService.java

```diff
@@ -0,0 +1,158 @@
+package org.mifos.connector.mockpaymentschema.service;
+
+import java.util.UUID;
+import org.mifos.connector.common.airtel.dto.AirtelEnquiryResponseDTO;
+import org.mifos.connector.common.airtel.dto.AirtelEnquiryResponseDataDTO;
+import org.mifos.connector.common.airtel.dto.AirtelEnquiryResponseDataTransactionDTO;
+import org.mifos.connector.common.airtel.dto.AirtelPaymentRequestDTO;
+import org.mifos.connector.common.airtel.dto.AirtelPaymentResponseDTO;
+import org.mifos.connector.common.airtel.dto.AirtelPaymentResponseDataDTO;
+import org.mifos.connector.common.airtel.dto.AirtelPaymentResponseDataTransactionDTO;
+import org.mifos.connector.common.airtel.dto.AirtelResponseStatusDTO;
+import org.mifos.connector.mockpaymentschema.schema.TransactionStatus;
+import org.mifos.connector.mockpaymentschema.schema.TransferStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AirtelMockService {
+
+    @Autowired
+    private org.mifos.connector.mockpaymentschema.service.SendCallbackService sendCallbackService;
+
+    @Value("${airtel.msisdnSuccessful}")
+    private String msisdnSuccessful;
+
+    @Value("${airtel.msisdnFailed}")
+    private String msisdnFailed;
+
+    @Value("${airtel.transactionIdSuccessful}")
+    private String transactionIdSuccessful;
+
+    @Value("${airtel.transactionIdFailed}")
+    private String transactionIdFailed;
+
+    @Value("${airtel.successResponseCode}")
+    private String successResponseCode;
+
+    @Value("${airtel.failedResponseCode}")
+    private String failedResponseCode;
+
+    @Value("${airtel.pendingResponsCode}")
+    private String pendingResponseCode;
+
+    @Value("${airtel.resultCode}")
+    private String resultCode;
+
+    public ResponseEntity<AirtelEnquiryResponseDTO> getTransactionStatus(String transactionId) {
+        String airtelMoneyId = UUID.randomUUID().toString().replace("-", "");
+        String message;
+        String status;
+        String code;
+        String responseCode;
+        boolean success;
+        HttpStatus httpStatus;
+
+        if (transactionId.equals(transactionIdSuccessful)) {
+            message = TransferStatus.SUCCESSFUL.name();
+            status = TransactionStatus.TS.name();
+            code = HttpStatus.OK.toString();
```

Comment:
The code now has the value `200 Ok` instead of `200` is that fine? Otherwise I think we have to create a seperate enum for response codes.